src/AzureClient/AzureClient.cs

```diff
@@ -572,10 +585,31 @@ public async Task<ExecutionResult> GetJobResultAsync(IChannel? channel, string j
                 //                 cancellation token support.
                 var request = WebRequest.Create(job.OutputDataUri);
                 using var responseStream = (await request.GetResponseAsync()).GetResponseStream();
-                return responseStream.ToHistogram(
-                    Logger, 
-                    isSimulatorOutput: this.ActiveTarget?.TargetId?.StartsWith(MicrosoftSimulator) ?? false)
-                .ToExecutionResult();
+                if (this.ActiveTarget?.TargetId?.StartsWith(MicrosoftSimulator) ?? false)
+                {
+                    var outputLines = ReadAllLines(responseStream).ToList();
```

Comment:
If I remember correctly, the idea was to show the same output as the simulator running locally. This includes all the output produced through the `Message` API, so I believe that trying to separate message from result is not needed.

Reply:
There is a subtle difference between how the Messages are displayed and how output is displayed, even when running the simulator locally. Output will be prefaced with a red 'Out[n]:' next to it visually differentiate it from the Messages. I am separating out the Messages from the output here to have the same effect.